test/syntax.js

```diff
@@ -57,63 +54,6 @@ describe('Azure swagger schema validation:', function () {
     });
   }).value();
 
-  describe('Azure composite swagger schema validation:', function () {
```

Comment:
@veronicagg So after looking at the way we were validating the `syntax` of composite schema was making sure the json of composite schema file adhere to the schema defined [here](https://raw.githubusercontent.com/Azure/autorest/master/schema/composite-swagger.json). But as we do not have concept of composite schema instead we have mow .md files, I am not sure whether we need a syntax validation. Thoughts? 

Regarding the line 43, it's a safety check that is dead now because we'll not have composite files at all. I can remove that if'd like or we can keep it for just in case.